project_docs/RELEASE_AND_SUPPORT.md

```diff
@@ -70,10 +72,20 @@ All nodes must be the same version. A topology may contain up to a maximum of 2
 
 Upgrades are supported from 1 release before the current stable release.
 
+> Example: A server running 1.2.0 may upgrade to version 1.3.0. A server running 1.2.0 may not
+> upgrade to 1.4.0.
```

Comment:
Should this be "1 minor version", since 1.2.1 -> 1.3.0 should be fine ?

Reply:
I think I'm assuming "release" here to mean minor version.